orttraining/orttraining/training_api/module.h

```diff
@@ -21,6 +21,8 @@ struct Parameter {
 
   // Return the mutable data.
   OrtValue& Data() { return data_; }
+  Status CopyTo(const DataTransferManager* data_transfer_manager, OrtValue& data) const;
+  Status CopyFrom(const OrtValue& data, const DataTransferManager* data_transfer_manager);
```

Comment:
is there a reason why the parameters naming order is not consistent?

Reply:
No particular reason. According to google style guide (https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs), inputs come in first and then outputs.
In this case, I can probably make the ordering the same for both functions.

Reply:
Let me know if you feel strongly about this. I'll change it then.

Reply:
It would be better if we followed the same patterns. Makes a huge difference.